librz/core/cmd_descs/cmd_interactive.yaml

```diff
@@ -0,0 +1,58 @@
+# SPDX-FileCopyrightText: 2024 RizinOrg <info@rizin.re>
+# SPDX-License-Identifier: LGPL-3.0-only
+---
+name: cmd_interactive
+commands:
+  - name: V
+    cname: interactive_visual
+    summary: Enter interactive visual mode
+    description: >
+      Use Rizin (mostly) without shell.
+      Scrolling disassembly, debugging, searching or graph views.
+      All with a few keyboard shortcuts.
+    args:
+      - name: visual-commands
+        type: RZ_CMD_ARG_TYPE_STRING
+        optional: true
+    details:
+      - name: Basic usage (in interactive visual mode)
+        entries:
+          - text: ""
+            arg_str: "Press: q"
+            comment: Quits the current view.
+          - text: ""
+            arg_str: "Press: ?"
+            comment: Opens the help.
+          - text: ""
+            arg_str: "Press: p"
+            comment: Switches between different printing modes (disassembly, hex view, etc.).
+          - text: ""
+            arg_str: "Press: <TAB>"
+            comment: Toggles different configurations of the current printing mode.
+      - name: Examples
```

Comment:
Add more information what the argument means - a sequence of the pressed keys in the visual mode. Also, it's better to have a few examples for those cases.

Reply:
Also, this is not full help. Either implement a full help or make `V?` corresponding to the one press of `?` in the `V` mode, and `V??` corresponding to two presses of `?` in the `V` mode.

Reply:
Then I go with `VH`/`VHH` for showing the help of the `V` mode. `V?` should be really just for the command.
But I'd like to have more details available to the user before they enter the V mode.
I remember I got confused my first time. It was the "How do I exit vim moment".

Reply:
Ok to have `VH` for the full help, but still examples section and explanation what the argument exactly is needed in the `V?` and `V??` output.